### PR TITLE
Clean Code - Comments - Explain Yourself in Code

### LINKED ISSUE #53: Book Club: Document our use of "Clean Code: Comments - Explain Yourself in Code"
Add documentation to the clean code comments document. Where applicable, add code examples to supplement your points in C# and TypeScript. All content should follow the patterns established in CODE_CONVENTIONS.MD

Notes:

Code should be self-commenting.
If you have a complex expression, consider moving it to a method or variable to self-document the expression.
We see this often, add code example.

### PR DESCRIPTION
**Pull request by brandongregoryscott**
Closes #53 Book Club: Document our use of "Clean Code: Comments - Explain Yourself in Code"

- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [-] Code builds cleanly (no _additional_ warnings or errors)
- [-] Manually tested
- [-] Automated tests are passing
- [-] No _decreases_ in automated test coverage
- [-] Documentation updated (readme, docs, comments, etc.)
- [-] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

**Review by jstevenson — APPROVED** [review]

**Review by myty — APPROVED** [review]

**Review by dylanjustice — APPROVED** [review]

File: clean-code/COMMENTS.md [hunk 1]
```diff
@@ -12,6 +12,69 @@ after their creation.
...
+        <React.Fragment>
+            {hasEditingAccess && <AdminContextMenu {...props} />}
+        </React.Fragment>
+    );
+}
+```
+
+**Ideal**
+
+Combine view model methods to capture this higher-level concept and reduce duplication.
```
Comment (KevinBusch) [inline clean-code/COMMENTS.md:56]: This comment is a little confusing to me.  What exactly is being combined?

Comment (brandongregoryscott) [reply clean-code/COMMENTS.md:56]: In this example, the previous example is expanded upon, combining `user.hasRole(...)` being called multiple times

Comment (KevinBusch) [reply clean-code/COMMENTS.md:56]: Oh, got it.  Makes sense.  I should have picked up on that.  You might consider expanding on this comment... something maybe like `Combine view model method executions into a new view model method to capture this higher-level concept and reduce duplication.`

**Review by KevinBusch — COMMENTED** [review]
All looks good!

**Pull request merged**